src/components/YAxis.js

```diff
@@ -17,7 +17,7 @@ const YAxis = ({ tickCt, scale, width }) => {
     return (
       <g key={i} transform={`translate(0, ${pos})`} className="tick">
         <line x2={width} y2="0" strokeOpacity={i === 0 ? '1' : '.25'} />
-        <text fill="#000" x="-32" y="0" dy=".32em">{fmt(v)}</text>
+        <text className="fs-14" fill="#000" x="-32" y="0" dy=".32em">{fmt(v)}</text>
```

Comment:
This is a totally fine approach, but we currently set the font size of the `.axis` elements in [`_viz.scss`](https://github.com/18F/crime-data-frontend/blob/master/sass/components/_viz.scss#L4). I think we should either refactor out the parts of `.axis` that we already have classes for or just adjust the `font-size` in `.axis`.

To my mind, the tradeoff is that this style of css leads to a lot more stuff in the HTML attributes and so for SVG it might be nice to keep the styling in component style classes (like `.axis`) and for the rest of the HTML we can use the simple classes approach.

On the other hand, it makes a ton of sense to keep things as consistent as possible across the code base which is a compelling argument for refactoring out the aspects of `.axis` that we have elsewhere.

@jpwentz which approach do you prefer?